src/test/frontend/components/query_table_test.cljs

```diff
@@ -13,73 +13,90 @@
          (= (mapv #(hash-map :block/content %) sorted-result)
             (#'query-table/sort-result (mapv #(hash-map :block/content %) result) sort-state))
          {:sort-desc? true :sort-by-column :block}
-         ["abc" "cde"]
-         ["cde" "abc"]
-
```

Comment:
I saw that whitespace was collapsed to avoid lint failure but this makes tests harder to maintain. The linter is per var so we can also just split up deftest vars. I can do this